extensions/integration-tests/src/notebook.util.ts

```diff
@@ -72,8 +72,11 @@ export const pythonKernelMetadata = {
 };
 
 export function writeNotebookToFile(pythonNotebook: azdata.nb.INotebookContents): vscode.Uri {
+	var path = require('path');
 	let notebookContentString = JSON.stringify(pythonNotebook);
-	let localFile = tempWrite.sync(notebookContentString, 'notebook.ipynb');
+	let localFile = path.join(process.cwd(), 'notebook' + Math.floor(Math.random() * 101) + '.ipynb');
```

Comment:
if this path already exists, what happens? I know it's really unlikely, but can we check if it exists first and if it does try to create it again with a different random number?